src/app/Setup.ts

```diff
@@ -95,6 +95,9 @@ export abstract class Setup {
     PageCollection.defaultMinCharToSearch = 3
     PageCollection.defaultCurrentPage = 0
     PageCollection.defaultPerPage = 10
+    PageCollection.defaultBeforeQueryAction = () => $.await.init('softQuery')
```

Comment:
As we speak, let's move this callbacks to vue-adap-table because PageCollection simply doesn't care about it. And later we will find a better solution, separating logic from vue-await to a another oure Typescript library.